crates/interpreter/src/exec.rs

```diff
@@ -220,11 +223,11 @@ impl<'m> Store<'m> {
         let inst = &self.insts[inst_id];
         let x = ptr.index();
         let t = inst.module.func_type(x);
-        let mut parser = inst.module.func(x);
+        let (mut parser, side_table) = inst.module.func(x);
         check_types(&t.params, &args)?;
         let mut locals = args;
         append_locals(&mut parser, &mut locals);
-        let frame = Frame::new(inst_id, t.results.len(), &[], locals);
+        let frame = Frame::new(inst_id, t.results.len(), &[], locals, side_table);
         Thread::new(parser, frame).run(self)
```

Comment:
```
error[E0502]: cannot borrow `*self` as mutable because it is also borrowed as immutable
   --> src/exec.rs:231:9
    |
212 |     pub fn invoke<'a>(
    |                   -- lifetime `'a` defined here
...
223 |         let inst = &self.insts[inst_id];
    |                     ---------- immutable borrow occurs here
...
231 |         Thread::new(parser, frame).run(self)
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |         |
    |         mutable borrow occurs here
    |         returning this value requires that `self.insts` is borrowed for `'a`
```